L1Trigger/L1TGlobal/interface/AXOL1TLCondition.h

```diff
@@ -24,6 +25,14 @@ namespace l1t {
   class L1Candidate;
   class GlobalBoard;
 
+  //template function for reading results
+  template <typename ResultType, typename LossType>
+  LossType readResult(hls4mlEmulator::Model& model) {
```

Comment:
I'd further suggest to place this function in to the `.cc` file, and inside anonymous namespace, in order to avoid any risk from symbol collisions.

Reply:
done! https://github.com/cms-sw/cmssw/pull/47552/commits/59888a0442342c4ebed1d63d3e59d70b1dd5d035